src/util/error.js

```diff
@@ -1,11 +1,3 @@
-import { isUndefined } from './is-type';
-import root from 'window-or-global';
-
 export default function error (message) {
-  const { DEBUG } = root;
-  if (!isUndefined(DEBUG) && DEBUG) {
-    console.error(message);
-  } else {
-    throw new Error(message);
-  }
+  throw new Error(message);
```

Comment:
I probably should have worded that differently. We'd always want it to throw in dev, but don't we want this to be a noop in prod (i.e. keep DEBUG)?